docs/source/conf.py

```diff
@@ -12,11 +12,7 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-import os
-import sys
-
-sys.path.insert(0, os.path.abspath(os.path.join("..", "..")))
-
+import configsuite
```

Comment:
Documentation will only build from project root anyway, as otherwise it will fail to fetch the version. Hence, I moved the import and removed the ugly environment injection.